### ISSUE TITLE
Updating docs for conda installation from conda-forge channel

### ISSUE THREAD

**Issue by sugatoray** [pytorch/captum#838]
## 📚 Documentation

Recently I packaged another library `ecco` for conda-forge channel. Since, `captum` is a dependency of ecco, I had to add captum to conda-forge as well (PR: https://github.com/conda-forge/staged-recipes/pull/17388). 

So, now you can also install captum from conda-forge channel.

```sh
conda install -c conda-forge captum
```

> I can send a PR to update the docs. Please let me know, if I should. :bulb:

**Comment by sugatoray** [comment]
![image](https://user-images.githubusercontent.com/10201242/148816453-4baa0eab-e2d6-4fc5-8b68-cf5dbcd349e3.png)

**Comment by sugatoray** [comment]
It will be nice to have someone from captum core maintainers also get added to the maintainer list for the conda-forge feedstock. If you could pick one or two names, that will be great. I will add them as co-maintainers of https://github.com/conda-forge/captum-feedstock.

**Comment by sugatoray** [comment]
cc: @vivekmig

**Comment by vivekmig** [comment]
Hi @sugatoray , if you can add @NarineK , @aobo-y , and me (@vivekmig)  as co-maintainers of the conda-forge feedstock, that would be great, thanks!

**Comment by aobo-y** [comment]
hi @sugatoray , we plan to release the next version around March 4th. Will let you know once it is ready on PyPl. Thx!

**Comment by sugatoray** [comment]
I will send a PR to update the docs here and add the new maintainers to the feedstock. So, please don't close this issue until that's done. 

cc: @vivekmig

**Comment by sugatoray** [comment]
Gentle ping: @vivekmig @aobo-y @NarineK Please see this PR:

- https://github.com/pytorch/captum/pull/849
